src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -20,7 +23,8 @@ public Integer oddSum(List<Integer> numbers) {
      * Вернуть количество вхождений объекта `element`</p>
      **/
     public Long elementCount(List<String> elements, String element) {
-        return null;
+
+        return elements.stream().filter(word -> word.equals(element)).count();
```

Comment:
```suggestion
        return elements.stream()
        .filter(word -> word.equals(element))
        .count();
```
давай писати стріми у відповідному стилі